src/annotations/components/annotationForm/AnnotationForm.tsx

```diff
@@ -137,6 +152,74 @@ export const AnnotationForm: FC<Props> = (props: Props) => {
     props.onClose()
   }
 
+  const isTimeInFuture = timeToCheck => {
```

Comment:
what is `timeToCheck`? is it a `Date`? a timestamp?

Reply:
number or string.  added comments.